src/main/java/uk/gov/justice/digital/hmpps/cmd/api/controllers/UserPreferenceController.kt

```diff
@@ -25,7 +25,7 @@ class UserPreferenceController(val userPreferenceService: UserPreferenceService)
         return ResponseEntity.ok(userPreferenceService.getuserPreference())
     }
 
-    @ApiOperation(value = "Update the snooze notification preference")
+    @ApiOperation(value = "Update the notification snooze preference for a user")
```

Comment:
"Update the notification snooze _until_ preference for a user"

Reply:
👍